requirements.txt

```diff
@@ -3,4 +3,4 @@ requests>=2.24.0
 retry>=0.9.0
 importlib-metadata>=4.11.4
 python-dotenv>=1.0.1
-qiskit>=1.0.0
+qiskit
```

Comment:
previously I had removed this entirely because it worked well when installed with either qiskit < 1.0 or > 1.0 "out of the box", will test the same here.

Reply:
Requirement already satisfied: qiskit-terra==0.46.2 in /home/jon/.pyenv/versions/qiskit-old/lib/python3.11/site-packages (from qiskit->qiskit-ionq==0.5.4) (0.46.2)

yup seems to work OK